src/components/SearchFilters/sampleFilters.json

```diff
@@ -2,8 +2,8 @@
   "selectedFilters": {
     "format": [],
     "language": [],
-    "dateAfter": "",
-    "dateBefore": "",
+    "dateFrom": "",
```

Comment:
appliedFilters.dateFrom is expected as an array. I see that these are slightly different searchFilters vs. appliedFilters, but is it possible to standardize the structure?

Reply:
Sorry missed this comment! I think if specific filter value are sometimes arrays, then they should always be arrays if possible.